### PR TITLE
Master analytic accounting kme

### PR DESCRIPTION
**Pull request by kme-odoo**
Task: https://www.odoo.com/web#id=21980&view_type=form&model=project.task&action=327&menu_id=4720
Pad: https://pad.odoo.com/p/r.eOyna05zFEkPQBf0

### CODE REVIEW

File: addons/account/views/account_analytic_view.xml [hunk 1]
```diff
@@ -67,8 +67,8 @@
         <menuitem
             name="Analytic Accounts"
             parent="menu_finance_entries"
-            action="analytic.action_account_analytic_chart"
-            id="menu_action_analytic_account_tree2"
+            action="analytic.action_analytic_account_form"
+            id="view_account_analytic_account_list"
```
Comment (mdi-odoo) [inline addons/account/views/account_analytic_view.xml]: Menu name should be in **model_name_menu** pattern .

File: addons/account/views/account_view.xml [hunk 2]
```diff
@@ -734,7 +734,7 @@
                                 <field name="account_id" domain="[('company_id', '=', company_id)]"/>
                                 <field name="amount_type"/>
                                 <field name="tax_id" domain="[('company_id', '=', company_id)]" widget="selection" />
-                                <field name="analytic_account_id" groups="analytic.group_analytic_accounting" domain="[('account_type', '=', 'normal'), ('company_id', '=', company_id)]"/>
+                                <field name="analytic_account_id" groups="analytic.group_analytic_accounting" domain="('company_id', '=', company_id)]"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_view.xml]: Missing opening **[** in domain .

File: addons/account/views/account_invoice_view.xml [hunk 3]
```diff
@@ -288,7 +288,7 @@
                                                 <field name="sequence" invisible="1"/>
                                                 <field name="manual" invisible="1"/>
                                                 <field name="account_id" groups="account.group_account_invoice"/>
-                                                <field name="account_analytic_id" domain="[('company_id', '=', parent.company_id), ('account_type', '=', 'normal')]" groups="analytic.group_analytic_accounting"/>
+                                                <field name="account_analytic_id" domain="[('company_id', '=', parent.company_id)" groups="analytic.group_analytic_accounting"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_invoice_view.xml]: Missing closing **]** in domain .

File: addons/account/views/account_invoice_view.xml [hunk 4]
```diff
@@ -265,7 +265,7 @@
                                     <field name="account_id" groups="account.group_account_user"
                                         domain="[('company_id', '=', parent.company_id), ('internal_type', '=', 'other')]"/>
                                     <field name="account_analytic_id" groups="analytic.group_analytic_accounting"
-                                        domain="[('company_id', '=', parent.company_id), ('account_type', '=', 'normal')]"/>
+                                        domain="[('company_id', '=', parent.company_id)"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_invoice_view.xml]: Missing closing **]** in domain .

File: addons/account/views/account_invoice_view.xml [hunk 5]
```diff
@@ -108,7 +108,7 @@
                         <field name="name"/>
                         <field name="sequence"/>
                         <field name="account_id" groups="account.group_account_user"/>
-                        <field name="account_analytic_id" domain="[('company_id', '=', parent.company_id), ('account_type', '=', 'normal')]" groups="analytic.group_analytic_accounting"/>
+                        <field name="account_analytic_id" domain="[('company_id', '=', parent.company_id)" groups="analytic.group_analytic_accounting"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_invoice_view.xml]: Same here .

File: addons/account/views/account_invoice_view.xml [hunk 6]
```diff
@@ -74,7 +74,7 @@
                         <group>
                             <field domain="[('company_id', '=', parent.company_id)]" name="account_id" groups="account.group_account_user"/>
                             <field name="invoice_line_tax_ids" context="{'type':parent.get('type')}" domain="[('type_tax_use','!=','none'),('company_id', '=', parent.company_id)]" widget="many2many_tags" options="{'no_create': True}"/>
-                            <field domain="[('company_id', '=', parent.company_id), ('account_type', '=', 'normal')]" name="account_analytic_id" groups="analytic.group_analytic_accounting"/>
+                            <field domain="[('company_id', '=', parent.company_id)" name="account_analytic_id" groups="analytic.group_analytic_accounting"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_invoice_view.xml]: Missing closing **]** in domain .

File: addons/account/views/account_invoice_view.xml [hunk 7]
```diff
@@ -403,7 +403,7 @@
                                     <field name="account_id" groups="account.group_account_user"
                                         domain="[('company_id', '=', parent.company_id), ('internal_type', '=', 'other')]"/>
                                     <field name="account_analytic_id" groups="analytic.group_analytic_accounting"
-                                        domain="[('company_id', '=', parent.company_id), ('account_type', '=', 'normal')]"/>
+                                        domain="[('company_id', '=', parent.company_id)"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_invoice_view.xml]: Same as above.

File: addons/account/views/account_view.xml [hunk 8]
```diff
@@ -756,7 +756,7 @@
                                 <field name="second_account_id" domain="[('company_id', '=', company_id)]"/>
                                 <field name="second_amount_type"/>
                                 <field name="second_tax_id" domain="[('company_id', '=', company_id)]" widget="selection"/>
-                                <field name="second_analytic_account_id" groups="analytic.group_analytic_accounting" domain="[('account_type', '=', 'normal'), ('company_id', '=', company_id)]"/>
+                                <field name="second_analytic_account_id" groups="analytic.group_analytic_accounting" domain="('company_id', '=', company_id)]"/>
```
Comment (mdi-odoo) [inline addons/account/views/account_view.xml]: Same as above .

File: addons/account_voucher/account_voucher_view.xml [hunk 9]
```diff
@@ -86,7 +86,7 @@
                         <group>
                             <field name="account_id"/>
                             <field name="tax_ids" widget="many2many_tags"/>
-                            <field name="account_analytic_id" domain="[('account_type','=','normal'), ('company_id', '=', company_id)]" groups="analytic.group_analytic_accounting"/>
+                            <field name="account_analytic_id" domain="('company_id', '=', company_id)]" groups="analytic.group_analytic_accounting"/>
```
Comment (mdi-odoo) [inline addons/account_voucher/account_voucher_view.xml]: Same as above .

File: addons/account_voucher/account_voucher_view.xml [hunk 10]
```diff
@@ -105,7 +105,7 @@
                         on_change="product_id_change(product_id, parent.partner_id, price_unit, company_id, parent.currency_id, parent.voucher_type)" />
                     <field name="name"/>
                     <field name="account_id"/>
-                    <field name="account_analytic_id" domain="[('account_type','=','normal'), ('company_id', '=', company_id)]" groups="analytic.group_analytic_accounting"/>
+                    <field name="account_analytic_id" domain="('company_id', '=', company_id)]" groups="analytic.group_analytic_accounting"/>
```
Comment (mdi-odoo) [inline addons/account_voucher/account_voucher_view.xml]: Same as above .

File: addons/account_voucher/voucher_sales_purchase_view.xml [hunk 11]
```diff
@@ -201,7 +201,7 @@
                                         on_change="product_id_change(product_id, parent.partner_id, price_unit, company_id, parent.currency_id, parent.voucher_type)" />
                                     <field name="name"/>
                                     <field name="account_id"/>
-                                    <field name="account_analytic_id" domain="[('account_type','=','normal'), ('company_id', '=', parent.company_id)]" groups="analytic.group_analytic_accounting"/>
+                                    <field name="account_analytic_id" domain="('company_id', '=', parent.company_id)]" groups="analytic.group_analytic_accounting"/>
```
Comment (mdi-odoo) [inline addons/account_voucher/voucher_sales_purchase_view.xml]: Same as above.

File: addons/account_voucher/voucher_sales_purchase_view.xml [hunk 12]
```diff
@@ -85,7 +85,7 @@
...
```
Comment (mdi-odoo) [inline addons/account_voucher/voucher_sales_purchase_view.xml]: Same as above .

File: addons/analytic/views/analytic_view.xml [hunk 13]
```diff
@@ -101,7 +101,7 @@
                                 <field name="currency_id" options="{'no_create': True}" groups="base.group_multi_currency"/>
                             </group>
                         </group>
-                        <notebook>
```
Comment (tivisse) [inline addons/analytic/views/analytic_view.xml]: Why creating an empty notebook? The notebook is overrided only once, so let's create it in the other view.

**Pull request merged**